tech.json

```diff
@@ -470,12 +470,20 @@
         {
           "name": "canBombHorizontally",
           "requires": [],
-          "note": "A horizontal bomb jump using a single bomb. Since it uses one bomb, it works with PBs as well.",
+          "note": [
+            "Place a single bomb then move Samus to the side to be boosted away from the explosion.",
+            "Hold forward while in the air to carry the momentum farther.",
+            "By moving forward just before the explosion the momentum will be greater, but this is hard to do.",
+            "Since it uses one bomb, this works with a Power Bomb or a Morph Ball Bomb."
+          ],
           "extensionTechs": [
             {
               "name": "canHBJ",
               "requires": [ "canBombHorizontally" ],
-              "note": "A horizontal bomb jump using two bombs. Longer range than single HBJ, much harder to execute, doesn't work with PBs."
+              "note": [
+                "A horizontal bomb jump(HBJ) using two bombs. Longer range than a single HBJ, much harder to execute, and doesn't work with PBs.",
```

Comment:
Should be a space before the parentheses.
I dont really like calling it a single HBJ, maybe it would be more clear to call it "bombing horizontally" or "canBombHorizontally" (is that what people normally refer to it as? I only hear HBJ being the double)